### PR TITLE
Use unused SQLCheckOperator.parameters in SQLCheckOperator.execute.

### PR DESCRIPTION
**Pull request by wjmolina**
The method `execute` of the class `SQLCheckOperator` is not using the property `parameters`.

### CODE REVIEW

**Review by uranusjr — APPROVED** [review]

**Comment by uranusjr** [comment]
Can we have a test for this please?

File: airflow/providers/common/sql/operators/sql.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 import ast
 import re
-from typing import TYPE_CHECKING, Any, Callable, Iterable, Mapping, NoReturn, Sequence, SupportsAbs
+from typing import TYPE_CHECKING, Any, Callable, Iterable, Mapping, NoReturn, Sequence, SupportsAbs, Union
```
Comment (uranusjr) [inline airflow/providers/common/sql/operators/sql.py]:
```suggestion
from typing import TYPE_CHECKING, Any, Callable, Iterable, Mapping, NoReturn, Sequence, SupportsAbs
```

File: airflow/providers/common/sql/operators/sql.py [hunk 2]
```diff
@@ -619,14 +619,21 @@ class SQLCheckOperator(BaseSQLOperator):
     ui_color = "#fff7e6"
 
     def __init__(
-        self, *, sql: str, conn_id: str | None = None, database: str | None = None, **kwargs
+        self,
+        *,
+        sql: str,
+        conn_id: str | None = None,
+        database: str | None = None,
+        parameters: Union[Iterable, Mapping] | None = None,
```
Comment (uranusjr) [inline airflow/providers/common/sql/operators/sql.py]:
```suggestion
        parameters: Iterable | Mapping | None = None,
```

**Review by uranusjr — CHANGES_REQUESTED** [review]

**Review by uranusjr — APPROVED** [review]

**Review by eladkal — APPROVED** [review]

File: airflow/providers/common/sql/operators/sql.py [hunk 3]
```diff
@@ -619,14 +619,21 @@ class SQLCheckOperator(BaseSQLOperator):
     ui_color = "#fff7e6"
 
     def __init__(
-        self, *, sql: str, conn_id: str | None = None, database: str | None = None, **kwargs
+        self,
+        *,
+        sql: str,
+        conn_id: str | None = None,
+        database: str | None = None,
+        parameters: Iterable | Mapping | None = None,
```
Comment (eladkal) [inline airflow/providers/common/sql/operators/sql.py:628]: need also to be added to docstring

**Review by eladkal — APPROVED** [review]
will merge when CI is green

**Review by potiuk — APPROVED** [review]

**Pull request merged**